File: app/src/main/java/com/wzq/sample/ui/recycler_single_network/detail/DetailFragment.java
package com.wzq.sample.ui.recycler_single_network.detail;

import android.os.Bundle;
import android.view.LayoutInflater;
import android.view.ViewGroup;

import com.wzq.mvvmsmart.base.BaseFragmentMVVM;
import com.wzq.sample.R;
import com.wzq.sample.databinding.FragmentDetailBinding;
import com.wzq.sample.bean.DemoBean;


/**
 * 详情界面
 */

public class DetailFragment extends BaseFragmentMVVM<FragmentDetailBinding, DetailViewModel> {

    private DemoBean.ItemsEntity entity;

    @Override
    public void initParam() {
        //获取列表传入的实体
        Bundle mBundle = getArguments();
        if (mBundle != null) {
            entity = mBundle.getParcelable("entity");
        }
    }

    @Override
    public int initContentView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return R.layout.fragment_detail;
    }

    @Override
    public int initVariableId() {
        return com.wzq.sample.BR.viewModel;
    }

    @Override
    public void initData() {
        viewModel.setDemoEntity(entity);
    }
}

File: app/src/main/java/com/wzq/sample/ui/recycler_network/detail/DetailFragment.kt
package com.wzq.sample.ui.recycler_network.detail

import android.os.Bundle
import android.view.LayoutInflater
import android.view.ViewGroup
import com.wzq.mvvmsmart.base.BaseFragmentMVVM
import com.wzq.sample.BR
import com.wzq.sample.R
import com.wzq.sample.bean.DemoBean.ItemsEntity
import com.wzq.sample.databinding.FragmentDetailBinding

/**
 * 详情界面
 */
class DetailFragment : BaseFragmentMVVM<FragmentDetailBinding, DetailViewModel>() {
    private var entity: ItemsEntity? = null
    override fun initParam() {
        //获取列表传入的实体
        val mBundle = arguments
        if (mBundle != null) {
            entity = mBundle.getParcelable("entity")
        }
    }

    override fun initContentView(inflater: LayoutInflater?, container: ViewGroup?, savedInstanceState: Bundle?): Int {
        return R.layout.fragment_detail
    }

    override fun initVariableId(): Int {
        return BR.viewModel
    }

    override fun initData() {
        viewModel.setDemoEntity(entity)
    }
}